client/src/editor/store/reducers/draftsReducer.js

```diff
@@ -347,6 +347,27 @@ export default function draftsReducer(drafts = initialState.drafts, action) {
         })
       );
     }
+    case types.SET_ADVENTURE_ENTRY_STORY_PART: {
+      const { firstPartName, firstPartKey, draftId } = action;
+      const firstPart = {
+        id: firstPartKey,
+        name: firstPartName,
+      };
+      const updatedDraft = {
+        ...drafts[draftId],
+        mainStory: {
+          ...drafts[draftId].mainStory,
+          firstPart,
```

Comment:
wait I'm confused now bc we have two example stories and I'm not sure what the change here really is. Can you update the example adventure found in /server/mock_data to accompany this so I can tell what the change is? While you're at it, make sure there's only one example (there is currently two) and that the one that remains is used in /server/db/seed.js